examples/puffin_profiler/src/main.rs

```diff
@@ -153,7 +153,7 @@ fn start_puffin_server() {
 
     match puffin_http::Server::new("127.0.0.1:8585") {
         Ok(puffin_server) => {
-            eprintln!("Run:  cargo install puffin_viewer && puffin_viewer --url 127.0.0.1:8585");
+            log::info!("Run:  cargo install puffin_viewer && puffin_viewer --url 127.0.0.1:8585");
```

Comment:
This means the user won't see the message unless they have `RUST_LOG=info` env-var set.

Either revert, or add this to before the call to `env_logger::init`:

```rs
// Show log messages by default:
let rust_log = std::env::var("RUST_LOG").unwrap_or_else(|_| "info".to_owned());
std::env::set_var("RUST_LOG", rust_log);
```